pkg/preprocessor/ottawau.py

```diff
@@ -85,7 +85,7 @@ def _parse_course_description(self, raw):
             )
         elif len(descriptions) == 0:
             return None
-        return descriptions[0].text
+        return descriptions[0].text.strip()
```

Comment:
There's a pesky newline at the beginning of every description, simplest fix seems to be just doing this